test/integration/image-component/typescript/pages/invalid.tsx

```diff
@@ -9,6 +9,13 @@ const Invalid = () => {
         id="no-width-or-height"
         src="https://via.placeholder.com/500"
       ></Image>
+      <Image
+        id="no-blur-data-url"
```

Comment:
Let's also add the corresponding assertion to the existing `should print error when invalid Image usage` test

Reply:
Only the first error thrown from `next/image` will be shown here since it won't render the next image tag and TypeScript errors aren't currently output in dev